examples/lines_bars_and_markers/eventcollection_demo.py

```diff
@@ -1,10 +1,10 @@
-"""
+r"""
 ====================
 EventCollection Demo
```

Comment:
There's a lot of Foo Demos; probably worth just doing a mass change as a single PR (together with deciding whether we want to style or to link the API name in the title as well (IIRC @QuLogic preferred styling but not linking in titles for the changelog, perhaps he can comment on that).

Reply:
I would handle Foo Demos separately. Im also very sceptical on using "Demo" at all.

Linking in titles usually doesn't render well because the <a> and <h*> CSS fight.

For a similar reason, I would not try to style section titles at all. They can appear as links in TOCs, and for gallery entries in particular their titles are used in the overview tiles as links as well.